src/test/java/ru/ayaz/UserTest.java

```diff
@@ -0,0 +1,54 @@
+package ru.ayaz;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by ayaz on 11/5/16.
+ */
+public class UserTest {
+
+    private User user;
+
+    @Before
+    public final void before(){ user = new User(); }
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    @Test
+    public void passingNull_setName() throws Exception {
+        exception.expect(IllegalArgumentException.class);
+        user.setNickName(null);
+    }
+
+    @Test
+    public void passingEmpty_setName() throws  Exception {
+        exception.expect(IllegalArgumentException.class);
```

Comment:
Could you switch the order of the action and expectation ?
It would be nice to have the structure:
given (you don't set up anything here)
when (action)
then (expectation)